clang/include/clang/3C/ConstraintVariables.h

```diff
@@ -226,9 +234,14 @@ class PointerVariableConstraint : public ConstraintVariable {
   // If for all U in arrSizes, any U -> (a,b) where a = O_SizedArray or
   // O_UnSizedArray, arrPresent is true.
   bool ArrPresent;
-  // Is there an itype associated with this constraint? If there is, how was it
-  // originally stored in the program?
+
+  // True if this variable has an itype in the original source code.
+  bool HasSrcItype;
```

Comment:
Rename `SrcHasItype` ? The word "source" could mean "source file" but also "source to sink"; I infer the latter (wrong) interpretation when I see `SrcItype`. Also matches the getter, above.